azkaban-web-server/src/main/java/azkaban/webapp/ExecutionLogsCleaner.java

```diff
@@ -29,6 +29,10 @@
    private static final long DEFAULT_LOG_CLEANUP_INTERVAL_SECONDS = 60 * 60;
    private long cleanupIntervalInSeconds;
 
+   // 1000 records
```

Comment:
can be removed, this looks implicit from variable name

Reply:
Sounds good. Removing it.